proto/firecracker.proto

```diff
@@ -135,3 +137,44 @@ message JailerConfig {
     DriveExposePolicy DriveExposePolicy = 7;
 }
 
+message UpdateBalloonRequest {
+    string VMID = 1;
+    int64 AmountMb = 2;
+}
+
+message GetBalloonConfigRequest{
+    string VMID = 1;
+}
+
+message GetBalloonConfigResponse{
+    FirecrackerBalloonDevice BalloonConfig = 1;
+}
+
+message GetBalloonStatsRequest {
+    string VMID = 1;
+}
+
+message GetBalloonStatsResponse{
+    int64 ActualMb = 1; // Actual amount of memory (in MB) the device is holding.
+	int64 ActualPages = 2; // Actual number of pages the device is holding.
```

Comment:
The indent here looks off.

Reply:
Yeah, I noticed it. I don't know how to fix it, since it look correctly in firecracker.proto. I tried various ways to fix it such as  leave a blank above the line 158, move the comment from line 158 to line 157, but none of them can fix it.

Reply:
Fixed it.